doc/security/use_cases/index.md

```diff
@@ -0,0 +1,185 @@
+These range from data center integrations, to embedded security applications such as security keys and smart cards.
+References to relevant specifications and certification targets are noted where use cases are backed by industry standards.
+
+
+## Platform Integrity Module
+
+An OpenTitan IC used as a Platform Integrity Module interposes between a platform's boot flash and its main boot devices such as the Baseboard Management Controller (BMC), the Platform Controller Hub (PCH) and the CPU.
+
+<img src="use_cases_fig1.svg" alt="Fig1" style="width: 300px;"/>
+
+Figure 1: Platform Integrity Module
+
+
+The Platform Integrity Module use case implements the following security properties:
+
+* Measure integrity of first boot firmware stages before bringing the boot devices out of reset accessing boot flash via SPI or similar interface.
+* Monitor resets and heartbeat of downstream boot devices.
+  Monitoring tasks are handled by OpenTitan as Interrupt Service Routines (ISRs), and are not expected to operate under real time constraints.
+* Enforce runtime boot device access policies, and manage A/B firmware updates for software stored in boot flash.
+  The OpenTitan to boot device interface is implemented on SPI or a similar interface.
+* Provides root key store and attestation flows as part of the platform integrity secure boot implementation.
+
+### Minimum Crypto Algorithm Requirements
+
+The current target for all crypto is at least 128-bit security strength.
+This is subject to change based on the implementation timeline of any given instantiation of OpenTitan.
+It is expected that a future implementation may be required to target a minimum of 192-bit or 256-bit security strength.
+
+* TRNG: Entropy source for chip identity generation and KDF operations.
+  In both cases, raw entropy is processed by a pseudorandom function before any key generation is done.
+* Hash Algorithms:
+  * SHA256: An approved hash algorithm with approximately the same security strength as its strongest asymmetric algorithm.
+* Asymmetric Key Algorithms:
+  * RSA-3072-bit: Secure boot signature verification.
+  * ECDSA P-256: Signature and verification for identity and attestation keys.
+* Symmetric Key Algorithms:
+  * HMAC-SHA256:  NIST FIPS 180-4 compliant. Used in integrity measurements for storage and in transit data as well as secure boot.
+  * AES: AES-CTR NIST 800-38A. Used to wrap keys and encrypt data stored in internal flash.
+
+
+### Provisioning Requirements
+
+Provisioning an OpenTitan device is performed in two steps:
+
+* Device Personalization: The device is initialized with a unique cryptographic identity endorsed by a Transit PKI which is only used to support initial Ownership Transfer.
+* Ownership Transfer: Ownership is assigned to a user that has the ability to run software on the device.
+  As Silicon Owner, the user can generate a cryptographic identity strongly associated to the hardware and the software version running on the device.
```

Comment:
Do we want to reference the definitions in `doc/security/logical_security_model`?